### PR TITLE
Add project token support

### PR DESCRIPTION
**Pull request by randomswdev**
Add support for project tokens, as desribed in issue https://github.com/xanzy/go-gitlab/issues/1098

### CODE REVIEW

File: project_access_tokens.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+	ID        	int   `json:"id"`
+	UserID      int   `json:"user_id"`
+	Name        string   `json:"name"`
+	Scopes      []string       `json:"scopes"`
+	ExpiresAt   *time.Time 	 `json:"expires_at"`
+	CreatedAt   *time.Time              `json:"created_at"`
+	Active      bool              `json:"active"`
+	Revoked    bool            `json:"revoked"`
+	Token  string            `json:"token"`
+}
```
Comment (profclems) [inline project_access_tokens.go:49]: The code should be gofmt'ed. This struct looks ugly

**Review by profclems — CHANGES_REQUESTED** [review]

File: project_access_tokens.go (hunk 1 above)
Comment (randomswdev) [reply project_access_tokens.go]: I formatted the code and added some logic to properly handle the timestamps GitLab returns when getting a token.

**Comment by randomswdev** [comment]
GitLab 13.10 has been release, so the library should now work fine. Is there anything additional I have to do for completing this pull request?

**Review by svanharmelen — APPROVED** [review]
Thanks @randomswdev 👍🏻  LGTM!

**Pull request merged**